### DISCUSSION TITLE
下载视频中加入生成aria2脚本且在脚本最后通过ffmpeg合并

### DISCUSSION THREAD

**Discussion by long45343** [the1812/Bilibili-Evolved#4594] [功能建议 / Ideas]
能否在aira2和ffmpeg均加入环境变量的情况下在下载视频时生成一个bat脚本，其中包含通过aira2下载视频并最后通过ffmpeg将音画合并后删除原来的视频和音频
ffmpeg合并脚本已经测试过：https://github.com/long45343/Bilibili-Evolved-downloaded-video-merge

**Comment by long45343** [comment]
想问下又理论上的可行性不😊

**Reply by the1812** [reply]
当然行啊 你仿照 aria2 写个插件装上就行了

**Reply by long45343** [reply]
那我看看怎么搞

**Comment by oxygenkun** [comment]
可以尝试用aria2提供的参数[文档](https://aria2.github.io/manual/en/html/aria2c.html#event-hook)
